purchases/src/main/kotlin/com/revenuecat/purchases/PurchaserInfo.kt

```diff
@@ -50,29 +52,53 @@ data class PurchaserInfo internal constructor(
     /**
      * @return Set of active subscription skus
      */
-    val activeSubscriptions: Set<String>
-        get() = activeIdentifiers(allExpirationDatesByProduct)
+    @IgnoredOnParcel
+    val activeSubscriptions: Set<String> by lazy {
+        activeIdentifiers(allExpirationDatesByProduct)
+    }
 
     /**
      * @return Set of purchased skus, active and inactive
      */
-    val allPurchasedSkus: Set<String>
-        get() = this.purchasedNonSubscriptionSkus + allExpirationDatesByProduct.keys
+    @IgnoredOnParcel
+    val allPurchasedSkus: Set<String> by lazy {
+        this.nonSubscriptionTransactions.keys + allExpirationDatesByProduct.keys
+    }
 
     /**
      * @return The latest expiration date of all purchased skus
      */
-    val latestExpirationDate: Date?
-        get() = allExpirationDatesByProduct.values.sortedBy { it }.takeUnless { it.isEmpty() }?.last()
+    @IgnoredOnParcel
+    val latestExpirationDate: Date? by lazy {
+        allExpirationDatesByProduct.values.sortedBy { it }.takeUnless { it.isEmpty() }?.last()
+    }
```

Comment:
made this lazy so it's only computed when accessed. The value will be stored too so it should be more efficient